include/lbann/layers/regularizers/selu_dropout.hpp

```diff
@@ -136,7 +136,7 @@ class selu_dropout : public regularizer_layer {
   void bp_compute() override {
     if (this->m_model->get_execution_mode() != execution_mode::training
         || m_keep_prob < 0.0f) {
-      El::LockedView(get_error_signals(), get_prev_error_signals());
+      El::Copy(get_prev_error_signals(), get_error_signals());
```

Comment:
Good catch, I'll make that happen

Reply:
On second thought, it takes some slight convolutions to make the activations or error signals a view, even more when it changes between owning data during training and being a view during evaluation. The SeLU stuff is exotic and not yet performance critical (especially since this optimization would only speed up evaluation), so I think simplicity is more important.